src/main/sensors/gyro.c

```diff
@@ -864,8 +864,8 @@ STATIC_UNIT_TESTED void performGyroCalibration(gyroSensor_t *gyroSensor, uint8_t
 
         if (isOnFinalGyroCalibrationCycle(&gyroSensor->calibration)) {
             const float stddev = devStandardDeviation(&gyroSensor->calibration.var[axis]);
-
-            DEBUG_SET(DEBUG_GYRO, DEBUG_GYRO_CALIBRATION, lrintf(stddev));
```

Comment:
I just want to confirm that the previous behavior here was either wrong, or un-expected. The 2nd argument is typically axis in per-axis debugging, or say 0/1 in pre-and-post Dynamic Notch debugging.

What would this have been with `DEBUG_GYRO`, `DEBUG_GYRO_CALIBRATION`? Is it just incorrect usage?

Reply:
The previous behaviour was wrong, as in field 3 of the debug array was overwritten 3 times, once for each axis. (This in itself is inefficient but not a problem, it's only Z axis value that will be logged in this case.)

Unfortunately, the behaviour introduced by this pull request is wrong too, since `DEBUG_GYRO_CALIBRATION` isn't an entry in `debugType_e`, but an index into `debug[]`. The `debug[]` array has 4 fields, so if it is used to record values for 3 axes (like in the `DEBUG_GYRO` case), field 3 isn't used and can be used to store an extra value, like `DEBUG_GYRO_CALIBRATION` is doing (in a clumsy way) in this case.